lib/api/src/entities/exception/exnref/mod.rs

```diff
@@ -0,0 +1,57 @@
+use std::any::Any;
+
+use crate::entities::store::{AsStoreMut, AsStoreRef};
+use crate::vm::{VMExceptionRef, VMExternRef};
+use crate::StoreRef;
+
+pub(crate) mod inner;
+pub(crate) use inner::*;
+
+#[derive(Debug, Clone, derive_more::From)]
+/// An opaque reference to some data. This reference can be passed through Wasm.
+pub struct ExceptionRef(pub(crate) RuntimeExceptionRef);
```

Comment:
This should be opaque (pub only for the crate)

Reply:
[6b716f5](https://github.com/wasmerio/wasmer/pull/5347/commits/6b716f57769ad520116ec0f997a8d8bee79978e9)